Sources/HTMLEntities/String+HTMLEntities.swift

```diff
@@ -0,0 +1,884 @@
+/*
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// Generated from the list of HTML4 entities here:
+/// https://www.w3.org/TR/html4/sgml/entities.html
+
+#if os(Linux)
+private let html4encode: [Character: String] = [
+    // ISO 8859-1 characters
+    // https://www.w3.org/TR/html4/sgml/entities.html#iso-88591
+    "\u{00A0}" : "&nbsp;",
```

Comment:
While 500 character lines are bad. We don't need only one pair per line either. Something reasonable in the middle would be great. Let' say 90 or so characters per line.